app/services/proofing/lexis_nexis/ddp/proofer.rb

```diff
@@ -63,6 +63,9 @@ def build_result_from_response(verification_response)
           result.add_error(:request_result, request_result) unless request_result == 'success'
           result.add_error(:review_status, review_status) unless review_status == 'pass'
 
+          result.success = !result.errors?
+          result.client = 'lexisnexis'
```

Comment:
would it be clearer to pass in client to the constructor? `DdpResult.new(client: 'lexisnexis')`

Reply:
Yeah, I didn't because of the `add_error` construction this thing has. That makes it tough to pass in `success` to the constructor so you'll end up doing something like this:

```ruby
body = verification_response.response_body

request_result = body['request_result']
review_status = body['review_status']

validate_review_status!(review_status)

result = Proofing::DdpResult.new(
  client: 'lexisnexis',
  response_body: body,
  transaction_id: body['request_id'],
  review_status: review_status,
)

result.add_error(:request_result, request_result) unless request_result == 'success'
result.add_error(:review_status, review_status) unless review_status == 'pass'

result.success = !result.errors?

result
```

That just looks kinda funny and discontinuous, but I am happy to change it if you think it is better this way.

Reply:
Yeah, I think the `#add_error` construction here is the primary divergence from the pattern we use elsewhere. I left it in to try and keep this as small as I could. I can go back around and get it. My primary goal here was to get the `#to_h` call to work correctly in `ResolutionProofingJob`.